sdk/src/types/api/core.rs

```diff
@@ -346,6 +364,7 @@ pub struct CongestionResponse {
     pub block_issuance_credits: i128,
 }
 
+// FIXME: can this response be removed ... seems like it's not exposed anymore!
```

Comment:
But we still use it internally? Maybe can be made pub(crate)

Reply:
no, also used by our integration tests, so has to be public.